config/etfuturum/blocksitems.cfg

```diff
@@ -0,0 +1,445 @@
+
+"misc blocks" {
+    #  [default: true]
+    B:enableBanners=true
+
+    # Enables log blocks with bark on all sides [default: true]
+    B:enableBarkLogs=true
+
+    # A solid, indestructible and invisible block. Can be seen when holding it in Creative mode. [default: true]
+    B:enableBarrier=true
+
+    # This must be on for the basalt deltas biome to generate unless Netherlicious is installed. [default: true]
+    B:enableBlackstone=true
+
+    #  [default: true]
+    B:enableChain=true
+
+    #  [default: true]
+    B:enableConcrete=true
+
+    # Copper sub-blocks and items. Disable copper but keep this on if you want the new copper items and blocks made of it, without the main ingot, ore or copper block itself. [default: true]
+    B:enableCopperSubItems=true
+
+    # Slabs for vanilla blocks: stone, mossy stone brick, mossy cobble, cut sandstone [default: true]
+    B:enableExtraVanillaSlabs=true
+
+    # Stairs for vanilla blocks: stone, mossy stone brick, mossy cobble [default: true]
+    B:enableExtraVanillaStairs=true
+
+    # Stairs for vanilla blocks: stone brick, mossy stone brick, sandstone, brick, nether brick [default: true]
+    B:enableExtraVanillaWalls=true
+
+    # Enables wood variant fences. [default: true]
+    B:enableFences=true
+
+    # Enables wood variant gates. [default: true]
+    B:enableGates=false
+
+    #  [default: true]
+    B:enableGlazedTerracotta=true
+
+    # Enables honey blocks, honeycomb blocks, honeycombs, and honey bottles. See entities.cfg for toggling bee nests, beehives, and bees. [default: true]
+    B:enableHoney=false
+
+    #  [default: true]
+    B:enableLantern=true
+
+    # Invisible light blocks. Only has a selection box when held, right click to change light level. Otherwise functionally identical to air and can be replaced by placing blocks into it. Invisible, but can be seen when holding it in Creative mode. [default: true]
+    B:enableLightBlock=true
+
+    #  [default: true]
+    B:enableQuartzBricks=true
+
+    # Note: Also enables cracked and chiseled nether bricks as they use this ID too [default: true]
+    B:enableRedNetherBricks=true
+
+    #  [default: true]
+    B:enableSmoothQuartz=true
+
+    #  [default: true]
+    B:enableSmoothSandStone=true
+
+    #  [default: true]
+    B:enableSmoothStone=true
+
+    # Enables stripped log blocks [default: true]
+    B:enableStrippedLogs=true
+}
+
+
+##########################################################################################################
+# misc items
+#--------------------------------------------------------------------------------------------------------#
+# Items that don't fit in any other category.
+##########################################################################################################
+
+"misc items" {
+    #  [default: true]
+    B:enableBeetroot=true
+
+    #  [default: true]
+    B:enableIronNugget=true
```

Comment:
already got iron nuggets. needs to be false

Reply:
Done.